src/debug/daccess/dacdbiimpl.cpp

```diff
@@ -3097,7 +3097,7 @@ TypeHandle DacDbiInterfaceImpl::GetExactFnPtrTypeHandle(ArgInfoList * pArgInfo)
 
     // convert the type information for each parameter to its corresponding type handle
     // and store it in the list
-    for (int i = 0; i < pArgInfo->Count(); i++)
+    for (unsigned int i = 0; i < pArgInfo->Count(); i++)
```

Comment:
`Count()` is defined as `int Count() const;` in `dacdbistructures.h`. Does gcc warn on line 3102?

Reply:
I did change it to unsigned int as part of this change.

Reply:
Ah I just realized that you changed it to `unsigned int` as part of this PR. I was comparing with `master`. 😄